src/test/groovy/org/mockito/release/internal/gradle/TemplateResolverTest.groovy

```diff
@@ -0,0 +1,81 @@
+package org.mockito.release.internal.gradle
+
+import spock.lang.Specification
+
+class TemplateResolverTest extends Specification {
+
+    def input = """
+    releasing {
+        gitHub.repository = "@gitHub.repository@"
+        gitHub.readOnlyAuthToken = "@gitHub.readOnlyAuthToken@"
+    }
+
+    allprojects {
+        plugins.withId("org.mockito.mockito-release-tools.bintray") {
+            bintray {
+                pkg {
+                    repo = '@bintray.pkg.repo@'
+                    licenses = @bintray.pkg.licenses@
+                    labels = @bintray.pkg.labels@
+                }
+            }
+        }
+    }
+"""
+
+    def output = """
+    releasing {
+        gitHub.repository = "mockito/mockito-release-tools-example"
+        gitHub.readOnlyAuthToken = "e7fe8fcdd6ffed5c38498c4c79b2a68e6f6ed1bb"
+    }
+
+    allprojects {
+        plugins.withId("org.mockito.mockito-release-tools.bintray") {
+            bintray {
+                pkg {
+                    repo = 'examples'
+                    licenses = ['MIT']
+                    labels = ['continuous delivery', 'release automation', 'shipkit']
+                }
+            }
+        }
+    }
+"""
+
+    def "should resolve template" (){
```

Comment:
I suspect that you can trim this test - already cover generation of the template. Here we might just test a simple template with few tokens to replace.